CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 4.2.7
+
+- Disable remote image insertion on public links @juliushaertl
+- Fixed file revision history behaviour in group folders @Raudius 
```

Comment:
I just merged the pending backport, so this needs a rebase and adding https://github.com/nextcloud/richdocuments/pull/2214 to the changelog :)

Reply:
Whoops, was a bit too eager to try out the release procedure :)

Rebasing done!

Reply:
No worries :) Just then we need also an entry on the changelog

Reply:
Fixed